### PR TITLE
 CRUD book details [Finishes #156073134]

### PR DESCRIPTION
**Pull request by rethes**
can view a dashboard with a summary of relevant information
can view all the books
can add a book and it's details
can edit book details
can update changed book details
can delete a book

### CODE REVIEW

**Comment by ipaullly** [comment]
good styling using div classes showing an a good grasp of bootstrap. you could have added comments, though. it will help your code's readability

File: Designs/UI/admin/addbook.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
+
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Add a Book</title>
+    <link rel="stylesheet" href="../assets/css/bootstrap.css">
+
+
```
Comment (Annettesunday) [inline Designs/UI/admin/addbook.html]: Remove all the white spaces

File: Designs/UI/admin/addbook.html [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+                </fieldset>
+            </form>
+
+
+        </div>
+    </div>
+
+
+
+
```
Comment (Annettesunday) [inline Designs/UI/admin/addbook.html]: Remove all the white spaces

File: Designs/UI/admin/allbook.html [hunk 3]
```diff
@@ -0,0 +1,139 @@
+
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>All the Books</title>
+    <link rel="stylesheet" href="../assets/css/bootstrap.css">
+
+
```
Comment (Annettesunday) [inline Designs/UI/admin/allbook.html]: Remove all the white spaces

**Comment by Annettesunday** [comment]
Generally looks good to me, but take care of the several white spaces first before you merge.

**Comment by Annettesunday** [comment]
Have descriptive commit messages

**Pull request merged**